src/StandAlone.cpp

```diff
@@ -46,7 +46,8 @@ int main() {
         aviTab.reset();
         env.reset();
     } catch (const std::exception &e) {
-        logger::error("Exception: %s", e.what());
+        std::cerr << "Exception: " << e.what() << std::endl;

```

Comment:
I think *replacing* the logger with an stderr output is a bit too much, there's actually a setting in the config whether AviTab should output to the console at all that's now overridden by this. Also, exceptions wouldn't appear in the log now. I think it would be better to either add a custom exception handler for the logger initialization and only output to stderr if that fails - or at least also use the logger here so that exceptions still appear in the log.

Reply:
Tried something else, but I don't like it too much to be honest.

Instead of adding `if (env->getConfig())` statements all over the place
to prevent segfaults when the config was not found, I recommend exiting
early when we cannot load a config. I don't do that in this PR, but I
can see that if we want to use the config often, the current
implementation seems a bit troublesome.